public/javascripts/application.js

```diff
@@ -1,2 +1,34 @@
-// Javascript Goes Here~
-//
+var Person = function(params){
+  this.name = params.name,
```

Comment:
Have you actually run this code?  The syntax is incorrect here... need ';' at the end of the line.